spec/puppet-lint/plugins/check_whitespace/arrow_alignment_spec.rb

```diff
@@ -332,6 +332,20 @@ class { 'lvs::base':
       end
     end
 
+    context 'multiline hash with opening brace on same line as first pair' do
+      let(:code) { "
+        foo { 'foo':
+          bar => [
+            { aa => bb,
+              c  => d},
+          ],
+        }
+      " }
+
+      it 'should not detect any problems' do
+        expect(problems).to have(0).problems
```

Comment:
Should we have a negative control to ensure that misalignment generates errors properly?